src/applications/mhv/secure-messaging/components/Dashboard/DashboardUnreadMessages.jsx

```diff
@@ -25,12 +25,12 @@ const DashboardUnreadMessages = props => {
 
       {folders !== undefined &&
         unreadCount > 0 && (
-          <h2 className="vads-u-font-size--h3">{`You have ${unreadCount} unread messages`}</h2>
+          <h2 className="vads-u-font-size--h3">{`You have ${unreadCount} unread messages in your inbox`}</h2>
```

Comment:
I believe the `unreadCount` is set by `unreadCountAllFolders`. Do we need to make a change in a calculation approach since this is changing to Inbox count only?

Reply:
or rather use Inbox folder object and derive unread count from there isntead?